packages/exchange/src/exchange/providers/google.py

```diff
@@ -24,6 +21,8 @@ class GoogleProvider(Provider):
     """Provides chat completions for models hosted by Google, including Gemini and other experimental models."""
 
     PROVIDER_NAME = "google"
+    BASE_URL_ENV_VAR = "GOOGLE_HOST"
+    BASE_URL_DEFAULT = "https://generativelanguage.googleapis.com/v1beta"
```

Comment:
does it make sense to have the constant at the top of the file just as people may want to quickly scan/change?

Reply:
So, these constants are accessed as class variables, so if we move to the top they'd no longer be that. I'm not sure a hack to work around this..